ip/iputil.go

```diff
@@ -48,6 +95,52 @@ func IsIPv4(ips ...interface{}) bool {
 	return true
 }
 
+// Check if an IP address is part of the list of internal IPs we have declared
+// checks for all ipv4 and ipv6 list
+func IsInternal(str string) bool {
+	_, err := newInternalRangeChecker()
+	if err != nil {
+		return false
+	}
+
+	if !IsIP(str) {
+		return false
+
+	}
+	IP := net.ParseIP(str)
+	for _, net := range rangeChecker.ipv4 {
+		if net.Contains(IP) {
+			return true
+		}
+	}
+	for _, net := range rangeChecker.ipv6 {
+		if net.Contains(IP) {
+			return true
+		}
+	}
+	return false
+}
+
+// Check if an IP address is part of the list of internal IPs we have declared
+func IsInternalIpv4(str string) bool {
```

Comment:
if the purpose here is to check if a cidr is an internal range, the function name should be renamed accordingly

Reply:
I used IsInIpv4List() instead